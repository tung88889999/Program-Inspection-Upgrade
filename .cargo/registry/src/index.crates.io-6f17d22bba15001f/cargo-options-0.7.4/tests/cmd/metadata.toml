bin.name = "cargo-mimic"
args = "metadata --help"
