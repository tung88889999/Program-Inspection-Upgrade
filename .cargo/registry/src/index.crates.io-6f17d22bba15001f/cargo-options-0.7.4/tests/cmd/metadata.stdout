Output the resolved dependencies of a package, the concrete used versions including overrides, in
machine-readable format

Usage: cargo-mimic[EXE] metadata [OPTIONS]

Options:
  -q, --quiet                     Do not print cargo log messages
  -v, --verbose...                Use verbose output (-vv very verbose/build.rs output)
      --filter-platform <TRIPLE>  Only include resolve dependencies matching the given target-triple
      --no-deps                   Output information only about the workspace members and don't
                                  fetch dependencies
      --manifest-path <PATH>      Path to Cargo.toml
      --format-version <VERSION>  Format version [possible values: 1]
      --color <WHEN>              Coloring: auto, always, never
      --config <KEY=VALUE>        Override a configuration value (unstable)
  -Z <FLAG>                       Unstable (nightly-only) flags to Cargo, see 'cargo -Z help' for
                                  details
  -h, --help                      Print help

Feature Selection:
  -F, --features <FEATURES>  Space or comma separated list of features to activate
      --all-features         Activate all available features
      --no-default-features  Do not activate the `default` feature

Manifest Options:
      --frozen   Require Cargo.lock and cache are up to date
      --locked   Require Cargo.lock is up to date
      --offline  Run without accessing the network

Run `cargo help metadata` for more detailed information.
